applications: nrf5340_audio: Buildprog, pristine and flashing issue (#20963)
nrfconnect/sdk-nrf: applications/nrf5340_audio/doc/building.rst

The script is located in the :file:`applications/nrf5340_audio/tools/buildprog` directory.

  .. note::

>      The :file:`buildprog.py` is an app-specific script for building and programming multiple kits and cores with various audio application configurations.

Reviewer: ```suggestion
     The :file:`buildprog.py` script is an app-specific script for building and programming multiple kits and cores with various audio application configurations.
```

The script is located in the :file:`applications/nrf5340_audio/tools/buildprog` directory.

  .. note::

>      The :file:`buildprog.py` is a convenience script for building and flashing multiple kits and cores with various audio application configurations. With time, we will progress towards using only standard tools for building and flashing kits. :file:`buildprog.py` will be deprecated in a future release.
> 

Reviewer 2: ```suggestion
     The :file:`buildprog.py` is an app-specific script for building and programming multiple kits and cores with various audio application configurations. The script will be deprecated in a future release. The audio applications will gradually shift only to using standard tools for building and programming development kits. 

```

The command can be run from any location, as long as the correct path to :file:`buildprog.py` is given.

The build files are saved in separate subdirectories in the :file:`applications/nrf5340_audio/tools/build` directory.

> The script creates a directory for each transport, device type, core and version combination.

Reviewer 2: ```suggestion
The script creates a directory for each transport, device type, core, and version combination.
```

The command can be run from any location, as long as the correct path to :file:`buildprog.py` is given.

The build files are saved in separate subdirectories in the :file:`applications/nrf5340_audio/tools/build` directory.
The script creates a directory for each transport, device type, core, and version combination.

> For example, when running the command above, the script creates the :file:`unicast/gateway/app/debug`, :file:`unicast/gateway/net/debug`, :file:`unicast/headset/app/debug`, :file:`unicast/headset/net/debug` and directories.

Reviewer: ```suggestion
For example, when running the command above, the script creates the :file:`unicast/gateway/app/debug`, :file:`unicast/gateway/net/debug`, :file:`unicast/headset/app/debug`, :file:`unicast/headset/net/debug` files and directories.
```

      * For headset device: ``-DCONFIG_AUDIO_DEV=1``
      * For gateway device: ``-DCONFIG_AUDIO_DEV=2``

>    #. Choose the configuration file for the device selected by using one of the following options:

Reviewer 2: Rephrased this line a bit in #20817 , so you might want to update accordingly.
